### PR TITLE
Add SubSquare integration proposal

### PR DESCRIPTION
**Pull request by wliyongfeng**

### CODE REVIEW

File: open_rfps/SubSquare_integration_proposal.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+- Deploy SORA subsquare implementation.
+
+The basic features of this implementation will include:
+
+- Democracy proposals/referendums visualization and discussion.
+- Council/Technical committee motion visualization and discussion.
+- Overview and discussions feature.
+
+## Current roadblocks and barriers to success
+None.
```
Comment (WRRicht3r) [inline open_rfps/SubSquare_integration_proposal.md:37]: Please include roadblocks and barriers to success here, they can be as simple as this implementation not existing yet, or as complex as meriting a detailed breakdown.

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

**Comment by MishoKanaria** [comment]
A clean governance platform is needed for SORA and I do like Subsquares UI/UX. I  suggest to add any roadblocks whether technical or not.

File: open_rfps/SubSquare_integration_proposal.md (hunk 1 above)
Comment (wliyongfeng) [reply open_rfps/SubSquare_integration_proposal.md]: Thanks, added ddfa8d067463b52fbc0f9a2edfebf287eabd405c. Tell me if it's still not enough.

**Comment by wliyongfeng** [comment]
> A clean governance platform is needed for SORA and I do like Subsquares UI/UX. I suggest to add any roadblocks whether technical or not.

Thanks, tried to add 2 items at ddfa8d067463b52fbc0f9a2edfebf287eabd405c.

**Comment by Nolan730** [comment]
We definitly need a tool like this. It definitly has better UI and better timeline then polkaassembly. Its better then forking polkaassembly for my opinion. I suggest we make a remark proposal and see if community wants it. After that Subsquare team can confirm is it possible to implement Sora or not.

**Comment by wliyongfeng** [comment]
There are some blocks current `polkadot.js + sora types` can not decode correctly. See [block #412](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fsora.api.onfinality.io%2Fpublic-ws#/explorer/query/412).

ea5EJjm84o

> After that Subsquare team can confirm is it possible to implement Sora or not.

@Nolan730 I think we can still implement SORA though the upper decode problems, but I hope there are developers who can solve the decode problems.

**Comment by Nolan730** [comment]
I assume some types changed over time and you get error because of it. Probably you should use different types for older blocks and i think Subscan handles it like that.  Subsquare team should be able to handle this too

**Review by WRRicht3r — APPROVED** [review]

**Comment by wliyongfeng** [comment]
> I assume some types changed over time and you get error because of it. Probably you should use different types for older blocks and i think Subscan handles it like that. Subsquare team should be able to handle this too

Hi, as I replied in the telegram, It will be perfect if SORA developers can make tweak the types file to make the polkadot apps show the history blocks correctly. All external apps/external developers will benefit a lot from this work.

**Comment by wliyongfeng** [comment]
I think sora [versioned types](https://github.com/sora-xor/sora2-substrate-js-library/blob/master/packages/type-definitions/src/versioned.ts#L5) maybe misconfigured, and we can correct this to make all history blocks can be decoded. How can I contact the developers, so maybe we can co-work to solve this?

**Review by MishoKanaria — APPROVED** [review]

**Comment by MishoKanaria** [comment]
> I think sora [versioned types](https://github.com/sora-xor/sora2-substrate-js-library/blob/master/packages/type-definitions/src/versioned.ts#L5) maybe misconfigured, and we can correct this to make all history blocks can be decoded. How can I contact the developers, so maybe we can co-work to solve this?

Please message https://t.me/soradevs

**Pull request closed without merging**